src/common/components/vanilla/button/button-positive.css

```diff
@@ -0,0 +1,29 @@
+$button-color: $success;
```

Comment:
I've created the new directory `src/common/components/vanilla` for components I am implementing with the latest Vanilla styles. In the past, we created components in `/src/common/component` (e.g. `/src/common/button`). I hoped this would mark the start of an effort to make a modular version of Vanilla that we can publish in npm.

We should move the existing `<Button>` component in here, but there seems to be no equivalent to the branded orange buttons currently in use.